deploy/compose/compose.yaml

```diff
@@ -134,3 +134,13 @@ services:
       - ./container_files/init-sso:/init-sso${SELINUX_VOLUME_OPTIONS}
     entrypoint: /usr/bin/bash
     command: /init-sso/init.sh
+
+  nginx:
```

Comment:
I would put this in the compose-walkers.yaml or separate compose-report.yaml if for some reason it can't be added to walkers

Reply:
ok